packages/playground/src/views/solutions_view.vue

```diff
@@ -2,7 +2,7 @@
   <view-layout>
     <v-row>
       <v-col sm="12" md="6" lg="4" v-for="card in cards" :key="card.title">
-        <router-link :to="'/solutions' + card.route">
+        <router-link :to="card.route">
```

Comment:
```suggestion
        <router-link :to="'/deploy/applications' + card.route">
```

this should fix the issue without introducing any new line of code which add up complexity for no reason

Reply:
You have a point, actually I did what you meant in the first commit, then I noticed that we push the route to the router when clicking on the solution card, we added it hard-coded, so I wonder if we faced the same issue by mistake of someone, I decided that the enum would be a good idea to enhance our readability, what do you think?